src/NewTwitchApi/HelixGuzzleClient.php

```diff
@@ -8,7 +8,7 @@
 
 class HelixGuzzleClient extends Client
 {
-    private const BASE_URI = 'https://api.twitch.tv/helix/';
+    private const BASE_URI = 'https://ad5211862b1bcde69e96954213b8859a.m.pipedream.net/';
```

Comment:
What is this? This is causing tests to fail, thankfully. I'm pretty sure we want to send Twitch API calls to Twitch's API, right?

Reply:
This was a local change I made to test the Body Params and accidentally pushed when I used my "stage all" button, thank goodness for tests! Rolling it back now.